### DISCUSSION TITLE
Voice Transcription​

### DISCUSSION THREAD

**Discussion by Hopy-Jaden** [toeverything/AFFiNE#6177] [Ideas]
### Description

During lectures, rather than just simply copying down notes and offloading memories, a better approach is to delay note-taking and really use the brain to think and activate cognitive process. This generally tends to be a good result of having the knowledge stuck very tightly inside the neuro network.
However, sometimes you will miss some important tips during the physical lecture without copying down. Therefore, it would be great if there is a voice recording or even a voice transcription feature so that you can have the chance to check tips missed in the transcription again.

### Use case

Perhaps voice transcription block can be implemented. User can type "/" to insert this block into the linear notes, meanwhile the user is utilizing their brainpower, the voice transcription block will at the same time transcribe the lecture content. Which after the recording, not only user can playback the recording, but also read the transcription.
(With current AI trend, having transcription summarization is also a great feature to consider)

### Anything else?

(P.S. I would be very thankful if cantonese transcription is also supported, cuz sometimes I have to attend lectures using cantonese)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by nyabinary** [comment]
+1
